### ISSUE TITLE
1.19.3 compatibility

### ISSUE THREAD

**Issue by tambry** [zabi94/ExtraAlchemy#151]
It would be nice to have a 1.19.3 release. 🙂

**Comment by zabi94** [comment]
Yes it would

**Comment by tambry** [comment]
I just noticed this is actually blocked by [lack of Patchouli for 1.19.3](https://github.com/VazkiiMods/Patchouli/issues/632), right?

I'm willing to pay a bounty for this.

**Comment by zabi94** [comment]
No, it's not that, I can ship without patchouli. 
It's that I genuinely have no time to dedicate to this atm. It will be ready whenever it will be ready
